fastapi/routing.py

```diff
@@ -356,7 +356,13 @@ def __init__(
         self.path = path
         self.endpoint = endpoint
         if isinstance(response_model, DefaultPlaceholder):
-            response_model = get_typed_return_annotation(endpoint)
+            return_annotation = get_typed_return_annotation(endpoint)
+            if inspect.isclass(return_annotation) and issubclass(
+                return_annotation, Response
+            ):
```

Comment:
It can be replaced with `lenient_issubclass`:
```suggestion
            if lenient_issubclass(return_annotation, Response):
```

Reply:
Thanks :+1: